care/emr/resources/scheduling/schedule/spec.py

```diff
@@ -77,6 +78,12 @@ def validate_for_slot_type(self):
                 end_time = datetime.datetime.combine(
                     datetime.datetime.now(tz=UTC).date(), availability.end_time
                 )
+                total_time = (end_time - start_time).total_seconds() / 60
```

Comment:
we are extracting seconds in line 87 and 88. can we refactor that to a variable and re-use that here instead? 🤔